Ginger/GingerCoreNET/Run/RunSetActions/RunSetActionAutomatedALMDefects.cs

```diff
@@ -42,6 +42,7 @@ public new static class Fields
             public static string SelectedDefectsProfileID = "SelectedDefectsProfileID";
             public static string DefectsOpeningModeForAll = "DefectsOpeningModeForAll";
             public static string DefectsOpeningModeForMarked = "DefectsOpeningModeForMarked";
+            public static string DefectsOpeningModeReviewOnly = "DefectsOpeningModeReviewOnly";
```

Comment:
Codacy found an issue: [Change the visibility of 'DefectsOpeningModeReviewOnly' or make it 'const' or 'readonly'.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6280018)